grafana/alerting-get-started/preprocessed.md

```diff
@@ -195,18 +194,15 @@ In this section, we define queries, expressions (used to manipulate the data), a
 
    In Prometheus, `vector(1)` is a special type of PromQL query that generates a constant vector. This is useful in testing and query manipulation, where you might need a constant value for calculations or comparisons. This query will allow you to create an alert rule that will be always firing.
 
-1. Remove the ‘B’ **Reduce expression** (click the bin icon). The Reduce expression comes by default, and in this case, it is not needed since the queried data is already reduced. Note that the Threshold expression is now your **Alert condition**.
+1. In the **Alert condition** section:
 
-1. In the ‘C’ **Threshold expression**:
+   - Keep `Last` as the value for the reducer function (`WHEN`), and `0` as the threshold value. This is the value above which the alert rule should trigger.
 
-   - Change the **Input** to **'A'** to select the data source.
-   - Enter `0` as the threshold value. This is the value above which the alert rule should trigger.
-
-1. Click **Preview** to run the queries.
+1. Click **Preview alert rule condition** to run the query.
 
    It should return a single sample with the value 1 at the current timestamp. And, since `1` is above `0`, the alert condition has been met, and the alert rule state is `Firing`.
 
-   {{< figure src="/media/docs/alerting/alerting-always-firing-alert.png" max-width="1200px" caption="A preview of a firing alert" >}}
+   {{< figure src="/media/docs/alerting/firing-alert-preview.png" max-width="1200px" caption="A preview of a firing alert" >}}
```

Comment:
Aaah are you running the transformer manually for this update @tonypowa. I believe @jdbaldry fixed this issue in the transformer. Would you mind pulling the latest version of the staging branch and rebuilding the transformer? 

If you would like you could also just let the automation tool pick these changes up if they are merged into the source repos

Reply:
>If you would like you could also just let the automation tool pick these changes up if they are merged into the source repos

ahh i already merged the changes into the source doc! 😓  I'm okay waiting for the continuous agent to kick off

Reply:
@Jayclifford345

Reply:
https://github.com/grafana/killercoda/pull/148/ I think

Reply:
I think we still have a problem with one of the images but I am merging to not block the pipeline. Its not a deal breaker and it was there from our initial commit

Reply:
What's up with the image? Anything I can do to fix it?

Reply:
@jdbaldry sorry false alarm my dude! I was reading the preprocessed markdown and thought the transformer was missing the images but all good in the actual stages